### ISSUE TITLE
where a plugin can contrôle mouse speed/frequency ?

### ISSUE THREAD

**Issue by djmisterjon** [cocopon/tweakpane#471]
sup friend,  on my side i contrôle mouse frequency update like this for some of my inputs.
```tsx
<button
	style={{ cursor: 'w-resize' }}
	onPointerDown={(e) => {
		const { min, max, step, speed } = metas ?? {}; // metas options of the input

		const sdx = (e.clientX - e.clientY);
		const pointermove = (e: PointerEvent) => {
			const dx = (e.clientX - e.clientY);
			let v = value + ((dx - sdx) * (speed ?? 0.5)); // speed of mouse 🟢
			if (step) v = Math.round(v / step) * step; // apply step if need 
			if (min !== undefined) v = Math.max(min, v); // apply clamping
			if (max !== undefined) v = Math.min(max, v);
			v = +v.toFixed(3);
			setValue(v);
			onChange?.(v);
		};
		document.addEventListener('pointermove', pointermove);
		document.addEventListener('pointerup', (e) => document.removeEventListener('pointermove', pointermove), { once: true });
		e.stopPropagation();
		e.preventDefault();
	}}
>
	{`${vkey}:`}
</button>
```

did you or can you expose a way to allow user hack mouse speed/frequency ?
1 pixel update is too face for lot of case.
can you guide how archive this with your api ?
related to this issu, thanks
https://github.com/0b5vr/tweakpane-plugin-rotation/issues/1

---

**expected behavior**
![8uuuj](https://user-images.githubusercontent.com/24865815/216791949-31987316-25fb-4408-b0c6-dddd95a675b9.gif)

**Comment by cocopon** [comment]
You mean you want to change a scale for dragging, right?

In the current spec it's estimated from its initial value or `step` option.
https://github.com/cocopon/tweakpane/blob/4f5175b8194dbd8b2e91e7bdcc50cef5f109db59/packages/core/src/common/util.ts#L118

**Comment by djmisterjon** [comment]
thanks

**Comment by cocopon** [comment]
Wait wait, I'm positive to add an option for changing the scale.

**Comment by djmisterjon** [comment]
> Wait wait, I'm positive to add an option for changing the scale.

ho oups, oki i will try done a monkey patch for my app and keep a eye on your commit 😊

**Comment by cocopon** [comment]
Added `keyScale` and `pointerScale` option to `NumberInputParams` in v4 beta.

```js
pane.addBinding(PARAMS, 'foo', {
  keyScale: 10,
  pointerScale: 1,
  ...
});
```
https://tweakpane.github.io/docs/api/interfaces/NumberInputParams.html

You can try it with `npm install tweakpane@beta` or downloading it manually: https://github.com/cocopon/tweakpane/releases/tag/4.0.0-beta.1

**Comment by djmisterjon** [comment]
hey thank i will give a try maybe this week

**Comment by cocopon** [comment]
Released in v4.0.0.
